createrelease.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
```

Comment:
Ha! It mostly was just that github's most mature api lib is in ruby, and I
had some old stuff that used it!

On Tuesday, March 24, 2015, aranhoide <notifications@github.com> wrote:

> In createrelease.rb
> <https://github.com/getlantern/flashlight-build/pull/142#discussion_r27095319>
> :
>
> > @@ -0,0 +1,37 @@
> > +#!/usr/bin/env ruby
>
> Ox flirting with Clojure and you with Ruby? You guys on a bet, or
> something? :P
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/flashlight-build/pull/142/files#r27095319>.
>


-- 
--
President
Brave New Software Project, Inc.
https://www.getlantern.org
pgp A998 2B6E EF1C 373E 723F A813 045D A255 901A FD89